.opencode/commands/career-ops-interview-intel.md

```
@@ -0,0 +1,12 @@
+---
+description: Load static interview guide for a specific company
+---
+
+Show the interview intelligence guide for the following company:
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)